java/client/test/org/openqa/selenium/remote/ErrorHandlerTest.java

```diff
@@ -326,6 +326,24 @@ public void testShouldStillTryToBuildWebDriverExceptionIfClassIsNotProvidedAndSt
       assertStackTracesEqual(expectedTrace, cause.getStackTrace());
     }
   }
+  
+    @SuppressWarnings("ThrowableInstanceNeverThrown")
+    @Test
+    public void testShoulNotBuildWebDriverExceptionIfClassOrStackTraceIsNull() {
```

Comment:
This is a test testShoulNotBuildWebDriverExceptionIfClass_and_StackTraceIsNull
See comment about && and ||, and let's add tests for other combinations.

Reply:
OK, updated